controllers/main.go

```diff
@@ -145,6 +143,11 @@ func NewMainController(params *chaincfg.Params, adminIPs []string,
 		return nil, err
 	}
 
+	ch := &CaptchaHandler{
+		ImgHeight: 127,
+		ImgWidth:  257,
```

Comment:
Pulled out of my ass and could be set in the config file, but that's not really needed now.